docs/licensing.md

```diff
@@ -0,0 +1,35 @@
+---
+slug: licensing
+append_help_link: true
+description: >-
+  Learn about licenses of r2c products.
+---
+
+# Licensing
+
+This document provides an overview of the licenses of important products created by r2c. See the list of the products and their licenses below:
+
+<dl>
+  <dt>Semgrep CLI</dt>
+    <dd>This project is open-sourced. Licensed under <a href="https://github.com/returntocorp/semgrep/blob/develop/LICENSE">LGPL 2.1</a>.</dd>
+  <dt>Semgrep Supply Chain</dt>
+    <dd>Proprietary. See <a href="https://semgrep.dev/terms">Terms of Service</a>.</dd>
+  <dt>Semgrep App</dt>
+    <dd>Proprietary. See <a href="https://semgrep.dev/terms">Terms of Service</a>.</dd>
+  <dt>DeepSemgrep</dt>
+    <dd>Proprietary. See <a href="https://semgrep.dev/terms">Terms of Service</a>.</dd>
+  <dt>Semgrep Registry</dt>
+    <dd>
+    <a href="https://semgrep.dev/explore">Semgrep Registry</a> is a collection of rules from various repositories:
+    <ul>
+    <li>Rules in the <a href="https://github.com/returntocorp/semgrep-rules">semgrep-rules</a> repository are licensed under Common Clause v1.0 or LGPL 2.1. Review <a href="https://github.com/returntocorp/semgrep-rules/blob/develop/LICENSE#L10">semgrep-rules license</a>.</li>
+    <li>Rules from third-party repositories in <a href="https://semgrep.dev/explore">Semgrep Registry</a> inherit the licenses of their source repositories. These licenses are displayed within the rule definition in the editor, for example, MIT license for dgryski’s rules.</li>
```

Comment:
How about linking to [this ruleset from Trail of Bits](https://semgrep.dev/p/trailofbits)?

Reply:
@pabloest great idea!

Reply:
Added!